s30157-MikolajMach/zajecia1/instalacja_debian_linux.txt

```diff
@@ -0,0 +1,109 @@
+Portów szeregowych nie potrzebuję więc nie muszę ich nawet włączać
+USB pozostawiam w ustawieniach domyślnych
+W udostępnionych folderach mogę przenosić pliki z maszyny fizycznej do maszyny wirtualnej
+ale narazie nie muszę nic przesyłać
+Interfejs użytkownika nie jest mi potrzebny
+Tak więc uruchamiam maszynę i zaczynam instalację
+Po uruchomieniu wyświetla mi się kilka opcji do wyboru
+Wybieram opcję instalacji zwykłej, ponieważ nie potrzebuję instalacji graficznej
+ani nie potrzebuję specjanych ustawień dla kolorów ani nie potrzebuję syntezatora mowy
+Oczywiście wybieram język angielski
+Muszę wybrać moją lokalizację, więc wybieram Other>Europe>Poland
+Następnie muszę wybrać w jaki sposób będzie działała klawiatura
+Wybieram United States i przechodzę do mapowania klawiszy i wybieram amerykański anglieski
+Następuje pobieranie danych z pliku iso który dołączyłem wcześniej
+Muszę teraz wpisać nazwę hosta nazywam to "debian12-demo"
+Potrzebuję hasła dla root-a, w moim przymadku będzie to "xyz"
+Następnie należy wpisać nazwę użytkownika, więc ustawiam ją na nazwę "user"
+Ponownie wpisuję "user" i ustawiam hasło na "xyz" i zatwierdzam wpisując je jeszcze raz
+Do wyboru partycji dysku wybieram opcję Guided bez LVM
+Wyświetla mi się tylko jeden dysk do wyboru: "SCSI3 (0, 0, 0) (sda) - 10.7 GB ATA VBOX HARDDISK"
+SCSI oznacza Small Computer Systems Interface czyli równoległa magistrala danych, 
+która służy do przesyłania danych między urządzeniami
+Po wyborze dysku mam do wyboru kolejne opcje tym razem dotyczą one tego jak chcę podzielić pliki na partycjach
+ponieważ nie posiadam jescze odpowiedniej wiedzy żeby się tym zajmować wybieram opcję pierwszą
+czyli tą w której wszystkie pliki będą znajdywały się na jednej partycji
+Mam możliwość usunięcia istniejących partycji i stworzenia własnych,
+dzięki czemu mógłbym każdą partycję ustawić według własnych preferencji
+Np systemy plików takich jak Ext2, ext3, ext 4 które są po prostu różnymi genreacjami systemu ext dzięki który służy do zapisywania
+Jfs czyli system plików z journalingiem
+Xfs czyli tak jak Jfs system plików z journalingiem tylko że z możliwością skalowania
+FAT16, Fat32 które są dość starymi systemami plików z ograniczeniami
+Btrfs który jest dość nowoczesnym systemem plików o dość dużej niezawodności związanej z mechanizmami kotroli integranlości plików
+Ja pozostawiam partycje które utworzyły się automatycznie
+Dostaję zapytanie dotyczące wybrania dodatkowej płyty, ale tego nie potrzebuję
+Następną rzeczą którą trzeba wybrać to na którym terytorium się znajdujemy żeby móc ściągać piki przez internet
+W moim przypadku to jest "Poland" i przechodzę dalej gdzie muszę wybrać miejsce z którego będę pobierać pliki
+Ja wybieram "ftp.task.gda.pl" ponieważ jest to najbliżej mnie i będzie dzieki temu jak najmniej opóźnień
+Nie posiadam proxy więc pomijam kolejny punkt
+Nie chcę brać udziału w ankiecie popularności pakietów więc w następnym punkcie wybieram "NO"
```

Comment:
Co to jest ankieta popularności pakietów?
Do czego jest używana?
Kiedy należy się zgodzić na udział w tej ankiecie?
Kiedy nie należy brać w niej udziału?
Czy można później wziąć w niej udział zmieniając swoją decyzję po zainstalowaniu systemu?